mtime.js

```diff
@@ -4,6 +4,10 @@ var async = require('async');
 function getFilesMtimes(files, concurrencyLimit, done) {
   var filesMtimes = {};
   async.eachLimit(files, concurrencyLimit, function(file, fileDone) {
+    if (typeof file !== 'string') {
+      return fileDone();
```

Comment:
I think this breaks a whole logic a this plugin, bacause the dependencies updated times won't be stored in `filesMtimes`. I would rather convert the Set to an Array in `index.js`

```
// collect info about input dependencies to compilation
  compiler.plugin('after-compile', function(compilation, afterCompileDone) {
    // get updated mtimes of file dependencies of compilation
    pluginContext.updateDependenciesMtimes(Array.from(compilation.fileDependencies), afterCompileDone);
  });
```